core/base-service/metric-helper.js

```diff
@@ -0,0 +1,45 @@
+'use strict'
+
+const { performance } = require('perf_hooks')
```

Comment:
i had no idea this existed in Node 👍

Reply:
I did a quick test that it exists in Node 9. 😁